drivers/client_access_control/app/controllers/client_access_control/history_controller.rb

```diff
@@ -40,6 +41,10 @@ def queue
     def pdf
       @user = User.setup_system_user
       current_user ||= @user # rubocop:disable Lint/UselessAssignment
+
+      # The user that requested the PDF generation. If job was kicked off from CAS, this is nil.
+      @requesting_user = User.find_by(id: params[:user_id]&.to_i) if params[:user_id].present?
```

Comment:
probably don't need both safe navigation and a check for present.  `find_by` should return nil if the `id` is blank....
```suggestion
      @requesting_user = User.find_by(id: params[:user_id]&.to_i)
```